mkdocs.yml

```diff
@@ -8,11 +8,11 @@ site_name: ACED-IDP Documentation
 
 nav:
 - index.md
-- getting-started.md
-- workflows/quick-start-guide.md
-- Requirements:
+#- getting-started.md
+- Getting Started:
   - requirements/gen3-client.md
   - requirements/gen3-tracker.md
+  - workflows/quick-start-guide.md
 - Use Cases & Workflows:
   - workflows/creating-project.md
   - Submitting Data:
```

Comment:
On some other pages..
- mention of deprecated g3t utilities in `workflows/restrict-access/` url
- I think there's some outdated info on adding metadata (eg [/workflows/metadata](https://aced-idp.github.io/workflows/metadata/)) where Task and Diagnostic Report are not currently supported in our submission ETL

Reply:
DIagnosticReport, restrict-access: `deprecated`
done

Reply:
Great thanks! One more thing...

> Additional resources can be added to the metadata files:
> - subjects (ResearchSubject, Patient)
> - specimens (Specimen)
> - assays (Task)
> - measurements (Observation)

 To my understanding, Task isn't being created or used in our current g3t setup. Assay info in SMMART for instance, is stored in the Observation with Specimen reference. Consider maybe condensing last two bullet points into one, eg
> - assays/measurements (Observations)
?

Reply:
```$ g3t add --help | grep task
  --task <value>: The task identifier, requires either patient or specimen
```